src/hooks/query-hooks.js

```diff
@@ -24,6 +24,25 @@ export function useJurorDraftQuery(jurorId) {
     : []
 }
 
+/**
+ * Queries if the juror has already recieve rewards
+ *
+ * @param {String} jurorId Address of the juror
+ * @returns {Object} All `jurorId` drafts not yet rewarded
```

Comment:
Maybe I'm missing something... but doesn't this just return a boolean? Should it actually be returning `data.juror.drafts` if that exists and is an array?

Reply:
Yes, the intention is to return a Boolean, the JS docs are off. Initially had implemented it to return the array of drafts already rewarded but then i thought it would be more straight forward to just tell if the array has an item which means that the juror has claimed rewards at some point.